Arduino_package/hardware/variants/datalogger_amb26/variant.h

```diff
@@ -96,10 +96,7 @@ extern void wait_for_debug(void);
 #define A6                                      AMB_D9  // PB_3 3.3V    CH6
 
 /* LED pin mapping */
-#define LED_G                                   AMB_D3  // Green
-#define LED_R                                   AMB_D2  // Red
-#define LED_B                                   AMB_D1  // Blue
-#define LED_BUILTIN                             LED_G   // Green

```

Comment:
I have changed LED_BUILTIN as NC for the LED_G no more exist.

Reply:
Please assign AMB_D3 to be used for LED_BUILTIN. It is used in my circuit. I did not notice the issue, sorry.

Reply:
> Please assign AMB_D3 to be used for LED_BUILTIN. It is used in my circuit. I did not notice the issue, sorry.

Lets refer to the latest commit 666c2a9f111fe682f98cfb71478ab6139c81bc9f